dependencies.yaml

```diff
@@ -115,6 +122,9 @@ dependencies:
       - output_types: [conda, requirements, pyproject]
         packages:
           - cython>=3.0.0
+      - output_types: [requirements, pyproject]
+        packages:
+          - librmm==24.6.*
```

Comment:
Why doesn't this have `-cu11` / `-cu12` suffixes and matrix entries?

Reply:
Good point, this is missing. I'll add it. FWIW since rmm is header-only technically we could get away with not suffixing at all, but I opted for consistency instead.

Reply:
We do also publish separate conda packages for CUDA 11 and CUDA 12, so that seems fine for now. Maybe something that we can revisit in the future.